src/components/postView/view/postDisplayView.js

```diff
@@ -50,8 +56,21 @@ const PostDisplayView = ({
   const [refreshing, setRefreshing] = useState(false);
   const [postBodyLoading, setPostBodyLoading] = useState(false);
   const [tags, setTags] = useState([]);
+  const [contentWidth, setContentWidth] = useState(WIDTH - 32);
+  const deviceOrientation = useSelector((state) => state.ui.deviceOrientation);
 
   // Component Life Cycles
+  useEffect(() => {
+    if (
+      deviceOrientation === orientations.LANDSCAPE_LEFT ||
+      deviceOrientation === orientations.LANDSCAPE_RIGHT
+    ) {
+      setContentWidth(Dimensions.get('window').height - 128);
```

Comment:
@aliseyalvi shouldn't this change since now we do not want to change contentWidth

Reply:
yes, will change that